notebooks/notebooks/paper/estimate-flops-katago-v2.ipynb

```diff
@@ -0,0 +1,662 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.    cdf = df.query(f"rows &lt; {STRONGEST_CONFIDENTLY_RATED_ROWS}")
</pre><p>nit: not clear to me what the variable name <code>cdf</code> means. My mind keeps thinking "cumulative density function" whenever I read it but that's not correct </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlignmentResearch/KataGoVisualizer/pull/64/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Ah cdf = current_df